kerrors/doc.go

```diff
@@ -14,5 +14,13 @@
 // You should have received a copy of the GNU Lesser General Public License
 // along with the klaytn library. If not, see <http://www.gnu.org/licenses/>.
 
-// Package kerrors contains universally accessible error variables used in Klaytn.
+/*
+Package kerrors contains universally accessible error variables used in Klaytn.
```

Comment:
@laggu Is it universally accessible? I'm not sure what this exactly means. At the moment, the package has 25 or so errors but it seems not enough. I think it would be better to describe the purpose of the package.